cli/packages/prisma-cli-core/src/commands/deploy/deploy.ts

```diff
@@ -382,7 +392,7 @@ ${chalk.gray(
         !this.flags['no-seed'] &&
         projectNew
       ) {
-        this.printHooks()
+        this.printHooks(noHook)
```

Comment:
It's a taste question, but speaks something against just having an `if(nooHook)` here instead of introducing an arg to `printHooks`?